packages/http-client/src/index.d.ts

```diff
@@ -0,0 +1,7 @@
+export type IncludeTypes = "tree" | "props" | "breakpoints";
```

Comment:
it looks like we need those types to be installable as a dependency

Reply:
Can you please elaborate on this?

Reply:
what I mean is that we need to make sure that we can update main repo without hardcoding things as much as possible, I just checked this isn't used yet, so I may not have a full context why this selection is here

Reply:
With the exported `loadProjectFromRest` function, you can include only data you would require from the REST API. By default all data is included.